bazel/linux/bundles.bzl

```diff
@@ -100,11 +126,22 @@ You can set up the bundle like this:
         "prepare_args": attr.string(
             doc = "Optional parameters to pass to the prepare_bin. Can use shell expansion.",
         ),
+        "init_cmds": attr.string_list(
+            doc = "Shell commands to run INSIDE THE VM to prepare the VM (before init_bin - optional)",
+        ),
+        "init_bin": attr.label(
+            doc = "Binary to run INSIDE THE VM to prepare the VM (optional)",
+            executable = True,
+            cfg = "exec",
+        ),
+        "init_args": attr.string(
```

Comment:
should this be a `string_list` instead of just `string`?

Reply:
I've used string instead of string_list throughout here in the hope of "clarifying" the escaping behavior. Maybe it's my brain, but if I see an argv passed as an array, I assume each element can contain spaces or weird characters, and it will be handled correctly (each element passed as is to the next process). I also assume that things like ~ or environment variables are NOT handled, as the array is passed as is to the next process.

This is NOT the case here: the arguments are passed via shell, if it was an array the element separation in the array would not match what the final command would see, as the shell will expand environment variables, and potentially interpret whitespace.

So I used a string. Somewhat similar to the old native python APIs where if you pass an array to subprocess it is propagated as is for argv, while a string is passed via `sh -c` and exposed to shell expansion.